spec/fileManipulation_spec.rb

```diff
@@ -0,0 +1,19 @@
+require '../DesafioLogParser/fileManipulation.rb'
```

Comment:
no ruby a gente usa `require` pra referenciar alguma gem, qnd vc precisar puxar um arquivo de dentro do projeto, a gente usa `require_relative`

```suggestion
require_relative '../DesafioLogParser/fileManipulation.rb'
```

Reply:
Sorry, descobri que funciona da forma que vc fez :D

https://www.thoughtco.com/requre-method-2908199

Reply:
haha tranquilo, eu deixei do jeito q vc sugeriu obrigado pelo review, aprendi bastante, principalmente das convenções